jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -22,9 +22,9 @@
     from jina.types.request import Request
 
 
-class RequestHandler:
+class MonitoringRequestMixin:
```

Comment:
Should we have this in a separate file, together with other monitoring related stuff? To have it all in one place and not clutter the main business logic anywhere?

Reply:
I don't think that it should go into other monitoring things are they are speadout everywhere in the codebase. Putting it out of the file is a good idea though